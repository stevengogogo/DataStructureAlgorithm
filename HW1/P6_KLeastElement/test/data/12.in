200 500
-6819 4712 4463 -4308 -5252 -3771 222 1358 -1039 290 460 -34 -7153 -4157 -6420 -384 885 -3409 6469 -1748 6287 5337 9139 8275 8195 -350 7338 2969 -9587 1923 -6356 2325 -567 -4883 -4063 1573 -5077 -6019 6169 -819 9261 -6174 -8783 2341 6432 6940 -8303 -4563 7282 4676 5790 -2884 -9369 -3161 9841 6104 9012 2868 -6149 2704 6426 150 3940 -846 -9807 7566 -6357 -4481 5887 3810 -7252 4889 -5030 6937 -1618 3967 2673 6143 -1760 -2950 3479 2090 9149 6861 -1415 6162 9521 -4492 -2519 1644 970 7654 3058 -4356 -8846 -4046 8544 6106 3827 -6524 -393 -8704 8007 -873 -7431 -6947 -7045 7652 4290 -8939 -3255 -3859 -5062 -5062 -9963 -245 3366 -7794 2459 3306 9538 6561 6945 -5721 -8913 6681 7864 -611 5026 -4183 -5652 -2347 5995 8836 3885 -8804 -375 -761 -6295 -7801 7980 -855 4467 -9676 -1801 -6652 -476 -8011 -6010 2952 1824 -837 -636 6550 3385 3175 -907 6065 9604 6830 7239 9978 -4456 7857 -106 2959 -4401 9736 -90 -972 -660 -2508 -9361 -1590 7066 -2931 7543 -5531 7733 -2933 -7821 -2866 279 9572 7103 5270 3301 -1071 -883 2128 9706 4772 -8951 7488 6634 5547 5337 2161 2811 -47
Query 146 179 8
Reverse 32 140
Query 117 153 11
Delete 70
Insert 78 6672
Delete 142
Insert 57 -5524
Delete 43
Query 109 123 9
Insert 199 -6266
Insert 184 2129
Insert 128 -5157
Delete 105
Query 35 180 105
Insert 2 -826
Query 165 188 21
Reverse 88 175
Reverse 106 167
Delete 194
Query 8 174 126
Delete 19
Insert 184 1592
Reverse 120 122
Insert 132 -598
Reverse 70 135
Delete 98
Query 26 52 21
Insert 78 4907
Insert 16 7087
Reverse 40 176
Insert 192 -8214
Delete 201
Delete 189
Insert 184 -8652
Reverse 48 57
Query 144 146 1
Reverse 92 101
Delete 165
Query 61 103 17
Query 37 115 67
Query 21 199 162
Query 66 139 71
Delete 11
Delete 127
Reverse 17 75
Insert 93 -30
Query 23 180 147
Delete 4
Delete 154
Query 11 44 16
Reverse 35 97
Insert 39 -583
Query 84 169 66
Reverse 193 197
Insert 180 7929
Query 42 176 17
Delete 159
Delete 7
Insert 80 8066
Insert 175 597
Insert 4 4297
Delete 113
Insert 98 -1324
Query 103 189 44
Reverse 197 201
Query 36 96 22
Insert 68 -3162
Insert 32 4075
Reverse 149 194
Query 119 133 4
Insert 189 -5806
Query 187 196 8
Query 162 174 5
Delete 124
Reverse 90 155
Delete 16
Query 184 186 2
Insert 119 5502
Insert 137 2028
Reverse 170 177
Insert 118 5829
Insert 40 -1258
Reverse 180 182
Reverse 37 157
Reverse 141 187
Reverse 131 172
Query 125 125 1
Reverse 63 137
Reverse 102 167
Delete 157
Delete 10
Delete 83
Insert 105 -4737
Delete 126
Delete 47
Delete 196
Delete 178
Delete 19
Reverse 146 174
Insert 166 -6936
Query 100 173 2
Query 54 125 10
Delete 199
Insert 36 6740
Query 131 160 16
Delete 34
Query 21 22 2
Reverse 15 123
Insert 180 3646
Reverse 1 124
Query 71 135 39
Insert 196 -7518
Reverse 46 80
Insert 8 -5039
Delete 178
Insert 42 -857
Query 108 131 22
Insert 76 2904
Query 160 161 2
Insert 167 -2436
Delete 164
Query 59 127 20
Query 115 177 29
Insert 68 -4631
Delete 43
Delete 100
Query 13 76 54
Reverse 6 45
Insert 142 3157
Delete 123
Insert 204 9801
Reverse 28 145
Insert 100 2145
Query 42 134 90
Query 89 128 7
Query 112 175 27
Insert 160 -5137
Insert 178 -8760
Reverse 29 185
Query 165 179 11
Delete 128
Insert 134 -504
Query 83 119 1
Insert 75 -3543
Insert 160 7067
Delete 141
Reverse 77 125
Insert 78 -7196
Query 160 199 22
Query 21 110 79
Query 59 184 114
Query 151 185 25
Query 177 179 1
Reverse 205 207
Delete 197
Delete 51
Query 74 185 72
Insert 124 -1792
Insert 29 -1092
Query 140 177 24
Insert 198 -2664
Delete 154
Reverse 184 208
Delete 197
Delete 116
Query 170 188 15
Reverse 151 181
Delete 103
Query 92 105 11
Reverse 31 158
Query 45 116 38
Query 4 152 91
Insert 115 -2860
Delete 126
Reverse 27 159
Reverse 162 187
Delete 184
Query 182 195 3
Insert 159 -6046
Query 47 194 24
Query 28 108 54
Query 111 118 3
Insert 193 -2467
Insert 186 622
Insert 135 -8447
Query 188 197 8
Reverse 98 184
Insert 28 -8172
Delete 29
Insert 115 1534
Reverse 210 210
Insert 124 -8144
Query 174 188 3
Reverse 54 156
Query 129 176 6
Insert 146 -8482
Delete 111
Delete 30
Reverse 131 150
Query 77 108 20
Query 96 182 40
Query 42 78 5
Delete 35
Insert 83 1611
Reverse 125 196
Query 54 100 25
Insert 94 3470
Query 153 171 13
Reverse 41 211
Delete 35
Delete 118
Reverse 173 176
Insert 153 2340
Delete 114
Query 124 192 13
Delete 161
Reverse 75 92
Query 83 92 8
Query 135 182 16
Reverse 96 147
Delete 135
Insert 185 396
Insert 190 7742
Reverse 2 28
Reverse 82 192
Delete 37
Delete 101
Reverse 60 164
Delete 196
Query 170 197 14
Query 19 72 37
Query 131 193 22
Query 58 158 86
Insert 121 -7796
Delete 141
Delete 180
Query 117 149 8
Delete 54
Reverse 128 169
Delete 74
Delete 59
Insert 2 7356
Insert 20 -4850
Insert 73 1277
Insert 53 -830
Reverse 36 107
Query 24 127 55
Reverse 143 164
Query 96 152 8
Query 144 166 16
Reverse 72 138
Reverse 189 206
Query 46 82 37
Query 7 94 72
Delete 169
Query 147 163 3
Query 37 187 119
Insert 1 2745
Reverse 177 180
Reverse 185 196
Insert 35 -8558
Delete 122
Insert 19 4373
Insert 52 -1056
Query 200 207 3
Insert 34 -706
Reverse 156 201
Query 203 209 2
Query 60 135 32
Delete 61
Delete 143
Reverse 38 165
Query 180 186 7
Delete 21
Delete 153
Query 6 47 36
Delete 144
Reverse 34 89
Insert 43 -9411
Delete 73
Delete 66
Insert 162 1151
Query 25 192 84
Reverse 110 150
Insert 168 -420
Query 66 158 22
Insert 15 -7269
Insert 142 -5578
Delete 31
Reverse 178 199
Delete 23
Query 172 174 3
Insert 133 3885
Insert 29 -9729
Reverse 17 39
Query 125 194 58
Insert 5 -7982
Query 151 176 16
Insert 70 -4851
Query 88 125 12
Insert 7 -3609
Query 32 76 41
Insert 112 3978
Insert 19 -7811
Query 119 123 1
Reverse 41 82
Insert 126 5028
Insert 86 -7727
Delete 37
Delete 30
Reverse 200 204
Reverse 155 198
Insert 213 6806
Insert 74 -7964
Query 166 209 43
Insert 103 9942
Insert 48 -8910
Insert 214 7515
Reverse 103 163
Reverse 197 212
Query 171 174 4
Insert 31 9612
Delete 81
Delete 165
Delete 5
Insert 198 -8960
Delete 163
Reverse 80 206
Delete 134
Insert 20 -4749
Delete 89
Delete 20
Insert 170 9235
Reverse 26 195
Insert 92 4911
Delete 147
Delete 89
Query 74 205 88
Reverse 202 211
Insert 109 -4642
Reverse 12 208
Query 56 121 32
Query 61 114 13
Insert 6 3156
Query 28 212 156
Reverse 175 176
Insert 126 2323
Reverse 108 145
Delete 31
Insert 95 9623
Query 183 206 4
Query 207 210 4
Insert 144 -5165
Query 39 69 14
Delete 7
Reverse 127 195
Delete 64
Query 150 164 10
Reverse 124 214
Query 46 119 42
Reverse 61 94
Delete 172
Query 203 207 5
Delete 203
Delete 203
Query 170 201 31
Insert 31 -3341
Reverse 14 69
Reverse 43 139
Insert 181 5517
Delete 11
Query 76 89 8
Reverse 116 178
Delete 24
Insert 105 -9167
Query 54 113 31
Insert 169 8959
Delete 156
Delete 53
Delete 190
Query 210 210 1
Query 54 71 8
Query 85 129 4
Reverse 106 143
Delete 209
Delete 104
Insert 7 -6573
Query 82 133 34
Delete 25
Delete 125
Delete 159
Reverse 126 185
Reverse 48 80
Delete 55
Insert 41 8629
Reverse 24 91
Delete 175
Query 54 146 28
Query 71 126 29
Reverse 95 156
Query 112 172 38
Query 83 136 27
Reverse 51 151
Reverse 103 106
Insert 142 2810
Insert 153 -7158
Insert 59 -5600
Insert 154 -2902
Query 32 161 48
Query 59 92 9
Reverse 139 171
Insert 15 9037
Delete 161
Query 63 103 25
Insert 10 -3904
Reverse 109 111
Insert 97 4904
Query 132 171 39
Insert 78 -1725
Delete 96
Reverse 164 191
Query 108 174 18
Insert 20 -5508
Delete 10
Query 98 196 40
Insert 4 3448
Insert 179 3398
Insert 10 344
Reverse 171 215
Query 158 213 4
Insert 19 6167
Delete 144
Query 165 177 4
Query 195 206 1
Query 110 183 3
Reverse 113 163
Insert 63 5034
Insert 160 -1119
Query 32 155 104
Query 33 133 69
Delete 105
Reverse 10 98
Query 103 197 19
Insert 85 -2935
Reverse 90 109
Insert 77 1569
Insert 76 9841
Insert 110 4808
Insert 150 -3677
Insert 164 -9459
Insert 207 9670
Reverse 219 219
Delete 216
Insert 160 6861
Query 213 215 1
Delete 140
Insert 118 5836
Insert 29 904
Reverse 53 148
Delete 160
Delete 5
Reverse 25 158
Reverse 22 165
Query 19 78 27
Insert 59 8306
Reverse 151 166
Insert 15 6427
Delete 113
Delete 53
Delete 112
Insert 46 7663
Query 68 122 52
Query 150 167 14
Reverse 83 156
Query 112 125 3
Insert 162 1284
Query 165 180 14
Insert 86 -8785
Query 111 161 14
Reverse 132 218
Insert 188 -8826
Query 48 95 40
Query 16 89 5
Reverse 3 207
Query 97 166 41
Delete 152
Insert 143 -8358
Reverse 38 150
Reverse 125 218
Delete 43
Query 155 221 8
Reverse 193 223
Reverse 30 158
Reverse 115 179
Query 28 187 50
Query 49 155 43
Reverse 154 195
Reverse 206 223
Delete 23
Reverse 223 223
