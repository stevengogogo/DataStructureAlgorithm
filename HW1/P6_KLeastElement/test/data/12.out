-2931
-4883
4676
5887
7543
6106
6561
-8303
-1590
7066
7864
9604
8195
-837
6104
-6524
1592
-660
-5030
2128
-2931
-3859
-3162
-9807
-6295
-384
1573
3301
7543
-5157
-4308
-660
7980
9149
-4631
-883
5887
-9587
1358
7654
7864
5829
-7252
3479
2459
6065
3885
279
2145
-7045
-6295
4075
-5039
2128
-6819
-6149
3940
-47
-7794
-47
3301
-6266
6469
-4046
-1039
4907
-3543
7338
-5030
1611
-7794
3301
9706
5995
-6652
6287
-4737
-5062
-1590
7864
6426
-476
-5030
7087
5887
2340
-5039
7742
-2950
7864
3301
2969
-47
-5157
6945
-7269
6550
-1258
1923
1534
9012
9521
-420
-611
9604
-2950
-8939
2325
-4883
-245
2704
-106
-3255
-5165
2704
9572
-6019
-2902
-7196
-873
-7196
-9411
6469
3885
-6266
-34
-846
9037
622
-2935
6806
-3162
5829
-8804
2459
-6420
-3162
-846
